XCUITests/AuthenticationTest.swift

```diff
@@ -41,11 +41,12 @@ class AuthenticationTest: BaseTestCase {
 
         //send app to background, and re-enter
         XCUIDevice.shared.press(.home)
+        sleep(1)
```

Comment:
I know this is not good, but how we can verify that the app is loaded again? the problem on Bitrise is that the app does not go completely to background and when it is launched, it is not code blocked... maybe wiating for an element in the springboard will make the trick instead of this...